### PR TITLE
updated spam2 line 53

### LINKED ISSUE #8234: Show Node Count in spam2/flags
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file https://github.com/publiclab/plots2/blob/main/app/views/spam2/_spam.html.erb#L53. Here remove && params[:action] != "_spam_flags" from the this line.

`  <% if params[:action] != "_spam_revisions" && params[:action] != "_spam_comments" && params[:action] != "_spam_flags" && params[:action] != "_spam_users"%>`

It should be something like this

` <% if params[:action] != "_spam_revisions" && params[:action] != "_spam_comments" && params[:action] != "_spam_users"%>
`

**To Test this** 
- Go to /spam2
- From Menu Dropdown go to Flag
- Check your change 

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
$DIFF
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by juliawithers**
Fixes #8234  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by keshavsethi** [comment]
@juliawithers can you Please reopen this PR and check if Travis still fails.
Thanks!!

**Comment by juliawithers** [comment]
@keshavsethi Hello, this still shows as an open pull request for me, and awaiting review. Should I close and resubmit? Not entirely sure how to handle this. Thanks!

**Comment by jywarren** [comment]
Hi @juliawithers and @keshavsethi -- to re-run the tests, you can close and reopen the PR and it re-triggers the Travis tests. Thanks a lot for your contribution, hope this helps! As a next step, if it doesn't resolve, could you open up the Travis results by clicking "Details" above, and copy in the errors that resulted? Then we can troubleshoot and help you figure out what's going on! 

Thanks again!!!

**Comment by keshavsethi** [comment]
@juliawithers  Awesome :tada: :tada: Thanks!!

**Comment by juliawithers** [comment]
@sagarpreet-chadha OK, done!

**Comment by cesswairimu** [comment]
@keshavsethi could you please review this when you get a min? I see a slight difference from what was outlined on the issue

File: app/views/spam2/_spam.html.erb [hunk 1]
```diff
@@ -58,9 +58,9 @@
...
     Info <i class="fa fa-caret-down text-secondary"></i>
   </a>
-  <% if params[:action] != "_spam_revisions" && params[:action] != "_spam_comments" && params[:action] != "_spam_flags" && params[:action] != "_spam_users"%>
-    <div class=" d-inline-flex"><a href="/spam2/filter/unmoderated/30"  class="badge badge-pill badge-light text-secondary mx-2 my-3"><%= Node.where(status: 4).length %> Unmoderated</a></div>
-    <div class=" d-inline-flex"><a href="/spam2/flags/filter/all/30" class="badge badge-pill badge-light text-secondary mx-2 my-3" ><%= Node.where('flag > ?', 0).length %> Flagged</a></div>
+  <% if params[:action] != "_spam_revisions" && params[:action] != "_spam_comments" && params[:action] != "_spam_users"%>
+    <div class=" d-inline-flex"><i class="fa fa-sticky-note text-info mt-3 ml-5" data-toggle="tooltip" data-placement="left" title="Unmoderated Nodes"></i><a href="/spam2/filter/unmoderated/30"  class="badge badge-pill badge-light text-info mx-2 my-3"><%= @node_unmoderated_count %></a></div>
```
Comment (keshavsethi) [inline app/views/spam2/_spam.html.erb:62]: @juliawithers  I have removed @node_unmoderated_count  in the controller. _Please keep these two lines_.

```
 <div class=" d-inline-flex"><a href="/spam2/filter/unmoderated/30"  class="badge badge-pill badge-light text-secondary mx-2 my-3"><%= Node.where(status: 4).length %> Unmoderated</a></div>
    <div class=" d-inline-flex"><a href="/spam2/flags/filter/all/30" class="badge badge-pill badge-light text-secondary mx-2 my-3" ><%= Node.where('flag > ?', 0).length %> Flagged</a></div>
```

Comment (juliawithers) [reply app/views/spam2/_spam.html.erb]: @keshavsethi I hope I understood correctly - I have replaced the lines. And I'm not entirely sure I did this the right way either... I have made the changes on my branch, but I'm unsure how to show the changes here in this pull request.

**Comment by cesswairimu** [comment]
Hi @juliawithers if you commit and push the changes to your branch `julia-edit` the changes will appear here..feel free let us know if we can help further thanks

**Comment by cesswairimu** [comment]
This looked okay now @keshavsethi I see you have a pending review. Kindly review once more when you get a chance. Thanks

**Comment by cesswairimu** [comment]
Hey @keshavsethi kindly review this when you get a chance...I also see conflicting line, not sure which one to keep, if you could solve the conflict too it would be super. Thanks

**Comment by cesswairimu** [comment]
Hi @juliawithers , seems the issue you were fixing in this PR was closed..Sorry for any inconveniences caused. Feel free to ping me if you need help finding another issue to work on..Closing this. Thanks

**Pull request closed without merging**